ui/src/main/java/com/hyperwallet/android/ui/repository/RepositoryFactory.java

```diff
@@ -27,6 +27,9 @@
 
 package com.hyperwallet.android.ui.repository;
 
+import com.hyperwallet.android.ui.transfermethod.repository.TransferMethodRepository;
```

Comment:
The class RepositoryFactory.java  should be deprecated. We need to create a factory class (TransferMethodRepositoryFactory) for the TransferMethodRepository in the transfermethodrepository module. Same for the other repositories 
@vshcherbyna-epam this is applicable to you too

Reply:
Done